landscaper/common/generate/openapi/generate.go

```diff
@@ -109,40 +109,47 @@ func execute() error {
 		return fmt.Errorf("an output path has to be specified")
 	}
 
-	if len(rootPackage) == 0 {
+	if len(rootPackages) == 0 {
 		return fmt.Errorf("a package has to be specified")
 	}
 
-	if len(inputDirectory) == 0 {
+	if len(inputDirectories) == 0 {
 		return fmt.Errorf("an import directory has to be specified")
 	}
 
 	if len(rootDirectory) == 0 {
 		return fmt.Errorf("the project's root directory has to be specified")
 	}
 
-	fp, err := filepath.Abs(inputDirectory)
-	if err != nil {
-		return err
-	}
-	info, err := os.Stat(fp)
-	if err != nil {
-		return err
-	}
-	if !info.IsDir() {
-		return fmt.Errorf("import path is not a directory")
-	}
+	directories := strings.Split(inputDirectories, ",")
```

Comment:
Is it possible to use [`StringSliceVar()`](https://pkg.go.dev/github.com/spf13/pflag#StringSliceVar) as now we accept n directories? With `StringSliceVar()` we won't need anymore to do the parsing ourselves.